test/mock-connector.py

```diff
@@ -59,27 +81,34 @@ def msg_consume(args):
     channel.start_consuming()
 
 def msg_update(args):
+    msg = __parse_update_message(args.json_msg_file)
```

Comment:
```
with open(msg_file) as fd:
  channel.basic_publish(exchange=fog_exchange,
                          routing_key=KEY_UPDATE, body=fd.read())
```

Reply:
No need to handle the exception. Let the user know if the file not exists

Reply:
If I do that, I do not have a manner to know when I send the default message.